other/hell/devil

```diff
@@ -1,22 +1,21 @@
 **Devil** | Solo Killing - Hell Team
 __Basics__
 Each day, the Devil may choose a player to send the devil’s wager to. 
-The target may choose to either learn another player’s role, or attack another player, but sell their soul in the process. 
-The Devil has a variety of uses for souls. 
+The devil's wager recipient may choose one of four options.
+The Devil can use the hell team's pool of souls.
 When everyone is dead or soulless the Devil wins.
 __Details__
-The Devil can't target soulless players. The wager is sent at the end of the day. 
-The Devil can anonymously communicate with the target & will be notified of the result of the wager - they will learn who was attacked/which person was inspected & what role the target saw. 
-
-If the player chooses to see another player’s role (immediate), it will be affected by weak disguises and obstructions. These effects are seen from the player’s perspective, not the Devil’s.
-If the player chooses to attack another player (soul sold: immediately; kill executed: end-night), then they will become soulless. The attacker is the Devil. 
-If the player chooses to attack the Devil the attack will not be executed, but they will not be informed of this. However, they still sell their soul to the Devil.
+At the start of the game, the Devil may choose a role as a weak disguise. If the Devil doesn’t choose a role, they will show up as a citizen.
 
-The Devil can use a soul to protect someone else from all attacks for the next night (day; immediate) or kill a player of their liking (day/night; immediate). 
-If the Devil is attacked at night and they have spare souls, they will be protected and a soul will be used up automatically. 
+The Devil is soulless and their soul is in the hell team's pool of souls.
```

Comment:
same as demon, maybe mention it affects the starting amount?

Reply:
Same as Demon

> The Devil is soulless and their soul is in the hell team's pool of souls.